libraries/AP_InertialSensor/AP_InertialSensor.h

```diff
@@ -202,6 +202,9 @@ class AP_InertialSensor : AP_AccelCal_Client
     uint8_t get_primary_accel(void) const { return _primary_accel; }
     uint8_t get_primary_gyro(void) const { return _primary_gyro; }
 
+    // Update the harmonic notch frequency
+    void update_harmonic_notch_freq_hz(float scaled_freq);
```

Comment:
This isn't called anywhere at the moment, but I gather it's going to be used soon?

Reply:
Indeed. Already flying well with it.